src/footer.scss

```diff
@@ -0,0 +1,33 @@
+// Small
+.wsu-sesquicentennial {
+    margin: 0 auto;
+    max-width: 73.125rem;
+    width: 100%;
+
+    &__link {
+        text-decoration: none;
+        text-align: center;
+        display: block;
+        padding: 20px;
+    }
+
+    &__background {
+        background: url('https://150.wayne.edu/images/150-footer-background-image.jpg') no-repeat center;
+        background-size: cover;
+    }
+
+    &__title {
+        max-width: 750px;
+        margin: 0 auto;
+        display: block;
+    }
+}
+
+// Large
+@media (min-width: 600px) {
+    .wsu-sesquicentennial {
+        &__link {
+            padding: 30px;
+        }
+    }
+}
```

Comment:
This is small, but can you make sure your editor is picking up the `.editorconfig`? Per https://github.com/waynestate/sesquicentennial-footer/blob/feature/sesquicentennial-footer/.editorconfig#L10 it should be inserting a newline at the end of this file.

Reply:
added a new line